SeaPublicWebsite/Models/EnergyEfficiency/YourRecommendationsViewModel.cs

```
@@ -24,23 +24,16 @@ public class YourRecommendationsViewModel : QuestionFlowViewModel
 
         public EnergyPriceCapInfo EnergyPriceCapInfo { get; set; }
 
-        public string GetEnergyPriceCapString()
+        public LocalizedHtmlString GetEnergyPriceCapString(IHtmlLocalizer<SharedResources> sharedLocalizer)
         {
             return EnergyPriceCapInfo switch
             {
-                EnergyPriceCapInfoNotParsed => "CostsAndSavingsAreEstimatesWarningTextUnknownPriceCapString",
-                EnergyPriceCapInfoParsed => "CostsAndSavingsAreEstimatesWarningTextString",
-                _ => "CostsAndSavingsAreEstimatesWarningTextNoPriceCapString"
-            };
-        }
-
-        public object[] GetEnergyPriceCapArgs(IHtmlLocalizer<SharedResources> sharedLocalizer)
-        {
-            return EnergyPriceCapInfo switch
-            {
-                EnergyPriceCapInfoNotParsed => [],
-                EnergyPriceCapInfoParsed e => [sharedLocalizer[$"Month{e.MonthIndex}String"].Value + " " + e.Year],
-                _ => []
+                EnergyPriceCapInfoNotParsed =>
+                    sharedLocalizer["CostsAndSavingsAreEstimatesWarningTextUnknownPriceCapString"],
+                EnergyPriceCapInfoParsed e =>
+                    sharedLocalizer["CostsAndSavingsAreEstimatesWarningTextString",
+                        sharedLocalizer[$"Month{e.MonthIndex}String"].Value + " " + e.Year],
```

This looks wrong?

Either way, it still has the issue around `sharedLocalizer[$"Month{e.MonthIndex}String"].Value` which will be brittle - it will be verbose but I honestly think it might be better here to have a map from `MonthIndex` to e.g. `JanuaryString` etc.. We'd at least then have safety around the localised string selector, and the ability to throw a really clear error if there's an index off somewhere, rather than a null reference.